README.md

```diff
@@ -6,8 +6,8 @@ Math Magician Application
 
 "Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:
 
-- Make simple calculations
-- Read a random math-related quote
+- Make simple calculations.
+- Read a random math-related quote.
```

Comment:
Kindly, remove part of the `README.md` file which is created along side with the CRA and use the provided template [here](https://github.com/microverseinc/readme-template) and edit the section for your app.